hitl/run.sh

```diff
@@ -39,10 +41,12 @@ sleep 30
 # Test pinging Booster. This exercises that:
 # * Booster's network is functioning as intended
 # * The Booster application is operational
-ping -c 5 -w 20 booster-hitl
+ping -c 5 -w 20 "${BOOSTER}"
+
+python3 -m miniconf -h
```

Comment:
Doesn't provide any more testing.

Reply:
Yeah, seems to be a leftover from testing.